src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -12,18 +11,17 @@ $weekday--margin-right: 10px;
   z-index: z(navbar-schedule);
   top: 0;
   left: 0;
-  width: 100%;
-  height: $NavBarSchedule--height;
+  height: 100%;
+  width: 40%;
```

Comment:
can you create a variable for that `$NavBarSchedule--width` and use for transform?

Reply:
fixed in 33957fc21429a0fe972b60cc5123bd1381ee81f5